arcade/pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "arcade-ai"
-version = "0.1.1"
```

Comment:
Aren't we at 0.1.1 already? https://github.com/ArcadeAI/arcade-ai/releases/tag/0.1.1